src/metabase/public_settings.clj

```diff
@@ -100,6 +100,9 @@
                   ;; is first read.
                   (do (setting/set-timestamp! :instance-creation (or (first-user-creation)
                                                                      (java-time/offset-date-time)))
+                      ;; Resolve analytics ns here to avoid circular dependency
+                      (classloader/require 'metabase.util.analytics)
+                      ((resolve 'metabase.util.analytics/track-event) :new_instance_created)
```

Comment:
Maybe you should just move `analytics-uuid` into the `util.analytics` namespace? Better to move code around and avoid circular dependencies and avoid runtime resolution entirely if there's a way to do so.